unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -399,9 +399,15 @@ viewByPrefix =
     )
 
 find :: InputPattern
-find =
+find = find' "find" False
+
+findGlobal :: InputPattern
+findGlobal = find' "find.global" True
+
+find' :: String -> Bool -> InputPattern
```

Comment:
Maybe a new type here instead of Bool?

Reply:
IsVerbose, and Global vs Local?
What was isVerbose again anyway? `ls` vs `find`?

Reply:
`isVerbose` is whether to include hashes in the results.

There was an unused `Bool` flag that I just repurposed for `find` vs `find.global`. I'm not actually sure what the intent of it was... since it was totally ignored in `HandleInput`. :)

I can do some `newtype`s though.

Reply:
Tried out a new type, did not love it, went with type alias.